src/home/steppers.vue

```diff
@@ -0,0 +1,70 @@
+<template lang="pug">
+#steppers
```

Comment:
Maybe we can give more life to this examples

![image](https://github.com/beercss/beercss/assets/43939417/f837b6d6-c702-400f-a411-367438ed2809)

Reply:
I think we need to be something like MD2. MD3 does not has steppers yet 😞

![image](https://github.com/beercss/beercss/assets/12979016/55d565e2-852e-4ab9-bd97-bf762aa5bdd8)
![image](https://github.com/beercss/beercss/assets/12979016/a1417847-a835-4b10-92ae-20c9df0d0e37)

To be compatible with dynamic theme and disabled buttons in MD3 we can have something like this:

![image](https://github.com/beercss/beercss/assets/12979016/bb418c3b-e04d-4937-9e4a-2df8f2429341)

Reply:
Well sounds good, can you add the vertical option too ?

Sorry for my bad example btw... had the impression that a divider must be colored when completed ( totally wrong ).